src/zkevm_specs/evm_circuit/execution/error_code_store.py

```diff
@@ -0,0 +1,40 @@
+from zkevm_specs.evm_circuit.table import RW
+from zkevm_specs.util.param import (
+    GAS_COST_CODE_DEPOSIT,
+    MAX_CODE_SIZE,
+    N_BYTES_MEMORY_ADDRESS,
+    N_BYTES_STACK,
+)
+from ...util import FQ
+from ..instruction import Instruction
+from ..opcode import Opcode
+from ...util import N_BYTES_GAS
+
+
+def error_code_store(instruction: Instruction):
+    # retrieve op code associated to oog constant error
+    opcode = instruction.opcode_lookup(True)
+    instruction.constrain_equal(opcode, Opcode.RETURN)
+
+    # the call must be coming from CREATE or CREATE2
+    instruction.constrain_equal(FQ(instruction.curr.is_create), FQ(1))
```

Comment:
I think the constraints at line 17 and 20 should be mentioned in MD document in the Constraints section as well?

Reply:
Great suggestion! fixed in [f7ebab4](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/475/commits/f7ebab40e672d960fc779230701f3f0ba6333700)